Can't build on FreeBSD due to lack of downstream support in rquickjs

I've been trying to get this running on FreeBSD but hit a brick wall.

First issue is that the latest rust package on FreeBSD is broken due to linking to the wrong libc, resulting in:

```
error: process didn't exit successfully: `rustc -vV` (exit status: 1)
--- stderr
ld-elf.so.1: /usr/local/bin/../lib/librustc_driver-a11f9aed9e23c8d3.so: Undefined symbol "_ZNSt3__122__libcpp_verbose_abortEPKcz"
```

This has been reported but not fixed: https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=282638

I used rustup instead, and got rustc/cargo installed ok. But building inv_sig_helper works up to rquickjs-sys, where the 0.6.0 build fails due to an error with a `JS_SetPropertyInternal` call:

```
error[E0061]: this function takes 5 arguments but 6 arguments were supplied
    --> /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/rquickjs-sys-0.6.0/src/inlines/common.rs:174:5
     |
174  |     JS_SetPropertyInternal(ctx, this_obj, prop, val, this_obj, JS_PROP_THROW as i32)
     |     ^^^^^^^^^^^^^^^^^^^^^^                           -------- unexpected argument #5 of type `JSValue`
```

The issue is that this call doesn't expect the `this_obj` reference. Bumping `Cargo.toml` to rquickjs-sys 0.6.2 gets past this error, but then fails here:

```
error: couldn't read /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/rquickjs-sys-0.6.2/src/bindings/x86_64-unknown-freebsd.rs: No such file or directory (os error 2)
  --> /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/rquickjs-sys-0.6.2/src/lib.rs:17:1
   |
17 | include!(concat!("bindings/", bindings_env!("TARGET"), ".rs"));
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Turns out they removed the FreeBSD bindings back in Jan 2023 due to not being able to generate bindings automatically via GitHub Actions: https://github.com/DelSkayn/rquickjs/issues/101

This leaves me rather stuck. I've commented on that issue but if anyone here has any ideas on how I might get around this please let me know.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -7,7 +7,6 @@ edition = "2021"
 
 [dependencies]
 regex = "1.10.4"
-rquickjs = {version = "0.6.0", features=["futures", "parallel"]}
 tokio = { version = "1.37.0", features = ["full", "net", "macros", "rt-multi-thread", "io-std", "io-util", "mio"] }
 reqwest = "0.12.4"
 lazy-regex = "3.1.0"
@@ -16,6 +15,13 @@ tokio-util = { version = "0.7.10", features=["futures-io", "futures-util", "code
 futures = "0.3.30"
 log = "0.4.22"
 env_logger = "0.11.5"
+
+[target.'cfg(not(target_os = "freebsd"))'.dependencies]
+rquickjs = {version = "0.6.0", features=["futures", "parallel"]}
+
+[target.'cfg(target_os = "freebsd")'.dependencies]
+rquickjs = {version = "0.6.2", features=["futures", "parallel", "bindgen"]}
+
 
 # Compilation optimizations for release builds
 # Increases compile time but typically produces a faster and smaller binary. Suitable for final releases but not for debug builds.

